READMEs/wallets.md

```diff
@@ -31,7 +31,7 @@ Here's how. In Python:
 
 ```python
 import web3
-private_key = web3.Web3().eth.account.create().privateKey
+private_key = web3.Web3().eth.account.create().key
```

Comment:
leaving this comment for 2 reasons:
1. please make sure this is the right replacement, and it works. Attach any relevant documentation that the old `privateKey` property is now simply `key`.
2. I'll tag it in the camel case ticket, since it's done here, separately :)

Reply:
alternately to the documentation, you can also attach a sample of trying out this part of the readme :D

Reply:
Here is the source from where I found out:
![Screenshot from 2021-06-08 10-08-54](https://user-images.githubusercontent.com/50501033/121141288-78cd0c00-c843-11eb-87a4-1a81272080e8.png)
And here is a sample of trying out that part:
![Uploading Screenshot from 2021-06-08 10-39-43.png…]()